### PR TITLE
FAT-7 - MODAUD: Create test plan to cover functionality with API Karate test

### PR DESCRIPTION
**Pull request by obozhko-folio**
[FAT-7](https://issues.folio.org/browse/FAT-7) - MODAUD: Create test plan to cover functionality with API Karate test

### CODE REVIEW

**Comment by obozhko-folio** [comment]
FAT-7-cucumber-report

File: mod-audit/src/main/resources/domain/mod-audit/features/undefined.feature [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+  @Undefined
+  Scenario: Delete audit record by id
+    * print 'undefined'
+
+  @Undefined
+  Scenario: Get all circulation logs
+    * print 'undefined'
+
+  @Undefined
```
Comment (alekGbuz) [inline mod-audit/src/main/resources/domain/mod-audit/features/undefined.feature:29]: Please remove tenants endpoint checking, as they provide some common logic.

Comment (obozhko-folio) [reply mod-audit/src/main/resources/domain/mod-audit/features/undefined.feature:9]: Removed, thank you.

File: mod-audit/src/test/java/org/folio/ModAuditApiTest.java [hunk 2]
```diff
@@ -29,4 +29,9 @@ public void tearDown() {
   public void testGetData() {
     runFeatureTest("audit-data");
```
Comment (alekGbuz) [inline mod-audit/src/test/java/org/folio/ModAuditApiTest.java:30]: Please remove audit-data feature, as this one is never used.

Comment (obozhko-folio) [reply mod-audit/src/test/java/org/folio/ModAuditApiTest.java]: Removed, thank you.

**Review by siarhei-charniak — APPROVED** [review]

**Review by alekGbuz — APPROVED** [review]

**Pull request merged**